### PR TITLE
Import previous project assets

### PR DESCRIPTION
**Pull request by FerchusGames**
Used assets from my previous project "TowerFrog" to get this project running quickly.

### CODE REVIEW

File: Towerfall/Assets/_Main/Scripts/Player/PlayerController.cs [hunk 1]
```diff
@@ -8,16 +8,23 @@ namespace Towerfall.Controllers
...
-        IObservable<float> Move { get; }
+        IObservable<float> Run { get; }
         IObservable<Vector2> DashStart { get; }
     }
 
     public interface IPlayerControllerEvent
     {
+        void SetPlayerControllerData(IPlayerControllerData playerControllerData);
+
         IObservable<Unit> DashEnd { get; }
```
Comment (Horothenic) [inline Towerfall/Assets/_Main/Scripts/Player/PlayerController.cs]: This should be in `IPlayerControllerInput`

**Review by Horothenic — CHANGES_REQUESTED** [review]

File: Towerfall/Assets/_Main/Scripts/Player/PlayerController.cs (hunk 1 above)
Comment (FerchusGames) [reply Towerfall/Assets/_Main/Scripts/Player/PlayerController.cs]: c0da1b18286545cda7795908f1264d35c13d5a1f

**Comment by FerchusGames** [comment]
I added a few commits to try to make the PlayerProperties more modular and to clean up the PlayerManager and PlayerController.

File: Towerfall/Assets/_Main/Scripts/Player/PlayerController.cs (hunk 1 above)
Comment (Horothenic) [reply Towerfall/Assets/_Main/Scripts/Player/PlayerController.cs]: You moved the wrong one, I meant `DashEnd`

File: Towerfall/Assets/_Main/Scripts/Player/PlayerManager.cs [hunk 2]
```diff
@@ -64,70 +67,109 @@ public partial class PlayerManager : ITickable
...
 
-            if ((IsJumping || IsJumpFalling) && Mathf.Abs(_playerControllerData.RigidbodyVelocity.y) < _playerProperties.JumpHangTimeThreshold)
+        private float GetAirAccelerationRate()
+        {
+            if (IsAccelerating())
             {
-                accelerationRate *= _playerProperties.JumpHangAccelerationMultiplier;
-                targetSpeed *= _playerProperties.JumpHangMaxSpeedMultiplier;
+                return _playerProperties.Movement.RunAccelerationRate 
+                       * _playerProperties.Movement.AirAccelerationMultiplier;
```
Comment (Horothenic) [inline Towerfall/Assets/_Main/Scripts/Player/PlayerManager.cs:147]: Why in 2 lines?

File: Towerfall/Assets/_Main/Scripts/Player/PlayerManager.cs [hunk 3]
```diff
@@ -64,70 +67,109 @@ public partial class PlayerManager : ITickable
...
             }
+            
+            return _playerProperties.Movement.RunDecelerationRate 
+                   * _playerProperties.Movement.AirDecelerationMultiplier;
```
Comment (Horothenic) [inline Towerfall/Assets/_Main/Scripts/Player/PlayerManager.cs:151]: Same

**Review by Horothenic — CHANGES_REQUESTED** [review]

File: Towerfall/Assets/_Main/Scripts/Player/PlayerController.cs (hunk 1 above)
Comment (FerchusGames) [reply Towerfall/Assets/_Main/Scripts/Player/PlayerController.cs]: 1637e433fda884bfa8b4a7739c561368b44cbd5a

**Review by Horothenic — APPROVED** [review]

**Pull request merged**